frontend/src/modules/Core/Components/ZingModal.tsx

```diff
@@ -58,20 +55,47 @@ const TempDialogTitle = (props: DialogTitleProps) => {
 }
 
 export const ZingModal = (props: ZingModalProps) => {
+  const TempDialog = styled(Dialog)(({ theme }) => ({
+    '&.MuiDialog-root *': {
+      fontFamily: 'Montserrat',
+    },
+    '& .MuiDialogContent-root': {
+      padding: theme.spacing(1, 4),
+      minWidth: props.containerWidth || 'fit-content',
+      minHeight: props.containerHeight || '100px',
+    },
+    '& .MuiDialogActions-root': {
+      padding: theme.spacing(2, 3, 3),
+    },
+    '& .MuiDialog-container': {
+      '& .MuiPaper-root': {
+        width: '100%',
+        minWidth: props.containerWidth || '500px',
+      },
+    },
+  }))
+
   return (
     <TempDialog
       onClose={props.onClose}
       open={props.open}
       BackdropProps={{
         sx: {
-          backgroundColor: 'rgba(0, 0, 0, 0.7)',
+          backgroundColor: 'rgba(0, 0, 0, 0.3)',
           '-webkit-tap-highlight-color': 'transparent',
         },
       }}
-      sx={props.sx}
+      // imo sx should go to DialogContent rather than the outermost container
```

Comment:
sx prop*

Reply:
@mluo24 what do you think

Reply:
This shouldn't matter anymore. I think the only thing that needs to be amended is the height and width of the container, which is already taken care of by the new props. Everything else can use the new components.